README.md

```diff
@@ -1,90 +1,141 @@
 # Research Computing Documentation Repo
 
+<!--- ================================================================== --->
```

Comment:
Don't add these headers, they get in the way when wanting to edit the file, and definitely not contributor info as that is contained in the GitHub history.